docs/en/observability/synthetics-configuration.asciidoc

```diff
@@ -224,9 +224,26 @@ By default, monitors are automatically retested if the monitor goes from "up" to
 If the result of the retest is also "down", an error will be created, and if configured, an alert sent.
 Then the monitor will resume running according to the defined schedule.
 Using `retestOnFailure` can reduce noise related to transient problems.
+
+`fields` (`object`)::
+A list of key-value pairs that will be sent with each monitor event.
+The `fields` are appended to {es} documents as `labels`,
```

Comment:
Not sure if we use this pattern, but we can reference the fields are basically the same as https://www.elastic.co/guide/en/beats/heartbeat/current/monitor-options.html#monitor-fields

Reply:
@vigneshshanmugam we aren't exposing complete API, fields are always appended under labels to make sure mappings are there and we also only support keyword value/pairs